src/main/java/com/redhat/jenkins/plugins/ci/CIMessageNotifier.java

```diff
@@ -89,30 +115,13 @@ public String getMessageContent() {
     public void setMessageContent(String messageContent) {
         this.messageContent = messageContent;
     }
-    public boolean isFailOnError() {
+    public boolean getFailOnError() {
         return failOnError;
     }
-
-    @DataBoundSetter
-    public void setFailOnError(boolean failOnError) {
+    public void setFailOnErrro(boolean failOnError) {
```

Comment:
still typo

Reply:
It would help if I actually committed my changes. Trying again.